src/main/java/org/reso/service/data/meta/EnumValueInfo.java

```diff
@@ -22,6 +23,12 @@ public String getValue()
       return value;
    }
 
+    public String getKey(String fieldName)
+    {
+
+        return LookupDefinition.getReverseLookupCache().get(fieldName).get(value);
```

Comment:
Nested accessors have a high probability of throwing null pointer errors. Should probably be an Optional with `.orElse()`.